docs/api/changelog.md

```diff
@@ -4,6 +4,15 @@
     .container hr{margin-top: 5px; }
 </style>
 
+## Sep 27, 2018
+
+- Add an app for `proposals`. ([@storymode7])
+  This adds following endpoints:
+  - /api/proposals allowing `POST`, `PATCH`, `GET`
+  - /api/proposals/:id which allows to view a specific proposals (`GET`)
```

Comment:
Get proposal details via `GET /api/proposals/:id`

Make all your changelog like this ^